.coveragerc

```diff
@@ -0,0 +1,14 @@
+[run]
+branch = True
+include =
+    fasjson_client/*
+
+
+[report]
+fail_under = 88
```

Comment:
Let's start with that but not wait too long to go to 100%, it's harder to do it after the fact.

Reply:
I just used an existing tox.ini file from another fedora-infra repository, that's what was being used there.

Reply:
OK, could you set it to 100% then? Thanks